### PR TITLE
[FEAT] Add test endpoint for retrieving a dummy restaurant card

### LINKED ISSUE #94: [FEAT] Add an endpoint to allow testing of inserting card data without hitting the live google API
This should allow frontend changes to pull fake data from the backend to test integration, without needing to go through the places API.

### PR DESCRIPTION
**Pull request by Alxjlv**
In order to allow the frontend to be decoupled from the data flow of the APIs used to retrieve the match data, this endpoint is being setup to allow the frontend to pull dummy data with the correct schema from the backend.

The schema should be filled out by the API team

Closes #94 

Added the supertest testing dependency

### CODE REVIEW

File: backend/src/routes/sessions.js [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
 const router = express.Router();
 
 // TODO: Handlers...
+// GET:
+router.get('/testCard', async (req, res) => {
+  const testCard = new Restaurant({
+    name: 'Lonestar',
+    location: 'Owen G Glen Building',
+    cuisine: 'European',
+    price: '5',
```
Comment (jemimahmartinez) [inline backend/src/routes/sessions.js:12]: I'm not too familiar with Mongo schema types but did you define the type for `cuisine` to be an array of strings in `backend/src/mongo/models/Restaurant.js`? If so, should it then be `cuisine: ['European']`? The same goes for `price` too?

Comment (Alxjlv) [reply backend/src/routes/sessions.js:12]: Good call - I'll check that out. I didn't see any specific definition of types in the backend, but I made the Restaurant schema for this PR so maybe I should add some strings in the schema definition

Comment (Alxjlv) [reply backend/src/routes/sessions.js:11]: @jemimahmartinez I checked the restaurant schema and the response from the server - it looks like when using new Restaurant({...}) it automatically converts fields into the correct format.

**Review by ayushjani — APPROVED** [review]
LGTM!

**Review by jemimahmartinez — APPROVED** [review]
Nice job 👌 works perfectly

**Pull request merged**